content-management-system/src/scripts/configuration.sql

```diff
@@ -0,0 +1,159 @@
+
+                         CONSTRAINT ADDRESS_ADDRESS_ID_PK PRIMARY KEY (ADDRESS_ID),
+                         CONSTRAINT ADDRESS_ADDRESS_TYPE_CHK CHECK (ADDRESS_TYPE IN ('PRESENT', 'PERMANENT')),
+                         CONSTRAINT ADDRESS_DIVISION_FK FOREIGN KEY (DIVISION_ID) REFERENCES DIVISION(DIVISION_ID),
+                         CONSTRAINT ADDRESS_DISTRICT_FK FOREIGN KEY (DISTRICT_ID) REFERENCES DISTRICT(DISTRICT_ID),
+                         CONSTRAINT ADDRESS_UPAZILA_FK FOREIGN KEY (UPAZILA_ID) REFERENCES UPAZILA(UPAZILA_ID)
+);
+
+-- Create Subject table
+CREATE TABLE SUBJECT (
+                         SUBJECT_ID BIGSERIAL,
+                         SUBJECT_NAME VARCHAR(255)  NOT NULL,
+                         NAME_LOCAL VARCHAR(255)  NOT NULL,
+                         CREATED_AT TIMESTAMPTZ NOT NULL,
+                         UPDATED_AT TIMESTAMPTZ NOT NULL,
+
+                         CONSTRAINT SUBJECT_SUBJECT_ID_PK PRIMARY KEY (SUBJECT_ID),
+                         CONSTRAINT SUBJECT_NAME_UK UNIQUE (SUBJECT_NAME),
+                         CONSTRAINT SUBJECT_NAME_LOCAL_UK UNIQUE (NAME_LOCAL)
+);
+
+-- INSERT DATA INTO SUBJECT TABLE
+INSERT INTO SUBJECT (SUBJECT_NAME, NAME_LOCAL, CREATED_AT, UPDATED_AT) VALUES
+                                                                           ('Bangla', 'বাংলা', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+                                                                           ('English', 'ইংরেজি', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+                                                                           ('General Math', 'সাধারিত গণিত', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+                                                                           ('Higher Math', 'উচ্চতর গণিত', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+                                                                           ('ICT', 'তথ্য ও যোগাযোগ প্রযুক্তি', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+                                                                           ('Social Science', 'সামাজিক বিজ্ঞান', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+                                                                           ('General Science', 'সাধারিত বিজ্ঞান', CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+
+
+-- Create ACADEMIC_INFO table
+CREATE TABLE ACADEMIC_INFO (
+                               ACADEMIC_INFO_ID BIGSERIAL,
+                               ACADEMIC_LEVEL VARCHAR(255) NOT NULL,
+                               GRADE DOUBLE PRECISION NOT NULL,
+                               CLASS VARCHAR(255) NOT NULL,
+                               CREATED_AT TIMESTAMPTZ NOT NULL,
+                               UPDATED_AT TIMESTAMPTZ NOT NULL,
+
+                               CONSTRAINT ACADEMIC_INFO_ACADEMIC_INFO_ID_PK PRIMARY KEY (ACADEMIC_INFO_ID)
+);
+
+-- Create join table ACADEMIC_INFO_SUBJECT for Many-to-Many relationship
+CREATE TABLE ACADEMIC_INFO_SUBJECT (
```

Comment:
Constraint missing. Add constraint for both primary and foreign key

Reply:
![Capture](https://github.com/sohag16030/CMS/assets/38885449/9a483c71-e432-4a6f-9058-9e250c4eaa0e)